pkg/reconciler/v1alpha1/revision/revision_test.go

```diff
@@ -723,23 +725,38 @@ func TestGlobalResyncOnConfigMapUpdate(t *testing.T) {
 		},
 	}}
 
+	// Setup the API machinery.
+	controllerConfig := getTestControllerConfig()
+	kubeClient, servingClient, _, _, controller, kubeInformer, servingInformer, cachingInformer, watcher, _ := newTestControllerWithConfig(t, controllerConfig)
+	stopCh := make(chan struct{})
+	var eg errgroup.Group
+	defer func() {
+		close(stopCh)
+		// Controller.Run never returns an error.
+		eg.Wait()
+	}()
+	eg.Go(func() error { return controller.Run(1, stopCh) })
```

Comment:
The controllers do this after the cache sync below.

Reply:
Moved below.